styles/colors-silicon-night.less

```diff
@@ -23,6 +23,7 @@
 @agate-component-text-color: var(--agate-component-text-color, #000000);
 @agate-focus-bg-color: var(--agate-focus-bg-color, @agate-accent);
 @agate-focus-color: var(--agate-focus-color, #ffffff);
+@agate-selected-color: var(--agate-selected-color, #000000);
```

Comment:
Do we need @agate-selected-color in colors-silicon-night skin? It seems that we don't use this variable in this file.

Reply:
you are right